src/commons/services/utils.js

```diff
@@ -116,19 +116,20 @@
 
         vm.getClusterDetails = function(clusterId) {
             clusterName = "Unassigned";
-            if($rootScope.clusterData !== null && clusterId !== "") {
+
+            if($rootScope.clusterData !== null && typeof clusterId !== "undefined") {
```

Comment:
is any specific reason for using "typeof" operator , because without that also we can get to know if any properties is undefined .